cliquet/initialization.py

```diff
@@ -395,7 +395,8 @@ def _prefixed_keys(key):
 
         # Fail if not only one is defined.
         defined = set(settings.keys()).intersection(set(keys))
-        distinct_values = set([settings[d] for d in defined])
+        distinct_values = set([str(settings[d]) for d in defined])
```

Comment:
Won't this cause a problem with settings that contain unicode values ?

Shouldn't we use something like `repr()` instead ?